src/dpq2/query.d

```diff
@@ -9,14 +9,24 @@ import std.exception: enforce;
 mixin template Queries()
 {
     /// Perform SQL query to DB
-    immutable (Answer) exec( string SQLcmd )
+    immutable (Answer) exec( string SQLcmd, ValueFormat resultFormat = ValueFormat.TEXT )
```

Comment:
I didn't noticed the difference in multiple statements, so I reverted this to not mix the behavior.
Main motivation for this was that it returns the values in text mode and it isn't possible to read the text formated values using for example `as!bool`.
So it would be nice to have the possibility to convert string values too - add new issue on this?

Reply:
> Main motivation for this was that it returns the values in text mode and it isn't possible to read the text formated values using for example as!bool.

Why not to use protocol version 3 (execParams) for that?

> So it would be nice to have the possibility to convert string values too - add new issue on this?

But libpq isn't contains this functional...

Reply:
I meant something like this:
```D
auto res = conn.exec("SELECT now()");
res[0][0].as!SysTime; // not possible
res[0][0].as!string; // not possible
res[0][0].format == ValueFormat.TEXT // not possible
only way is res[0][0].data and work it out ourself
```

Reply:
Too big amount of code should be written. And this code is already available in Postgres server source...
If anyone want to get binary data he can request binary result by execParams - this is more right way.

Reply:
Then right now the exec function is not much usefull for getting results, as they can't be read in some simple way.
At least there should be some way to get the resulting string so one can convert it on his own.
But let this be as it is now and and keep it from this PR. I reverted it to the old behavior - can be handled separately later.